### ISSUE TITLE
State why using Cloudflare is a problem

### ISSUE THREAD

**Issue by Rult** [azadi/esnicheck#1]
AFAIK, ESNI is still a [draft](https://datatracker.ietf.org/doc/draft-ietf-tls-esni/), so you can't set this up now without Cloudflare.

**Comment by azadi** [comment]
Yes, I agree, that's a good idea. I should be more clear what I mean why this is a problem -- we need more providers to be supporting ESNI and it cannot just be Cloudflare. I will try to improve the text.
